packages/contracts-bedrock/test/L2/GasPriceOracle.t.sol

```diff
@@ -351,4 +353,46 @@ contract GasPriceOracleFjordActive_Test is GasPriceOracle_Test {
         uint256 upperBound = gasPriceOracle.getL1FeeUpperBound(data.length);
         assertEq(upperBound, 111214);
     }
+
+    /// @dev Tests that `operatorFee` is 0 is Isthmus is not activated.
+    function test_getOperatorFee_succeeds() external view {
+        assertEq(gasPriceOracle.getOperatorFee(10), 0);
+    }
+}
+
+contract GasPriceOracleIsthmus_Test is GasPriceOracle_Test {
+    /// @dev Sets up the test suite.
+    function setUp() public virtual override {
+        l2Fork = Fork.ISTHMUS;
+        super.setUp();
+
+        bytes memory calldataPacked = Encoding.encodeSetL1BlockValuesIsthmus(
+            baseFeeScalar,
+            blobBaseFeeScalar,
+            sequenceNumber,
+            timestamp,
+            number,
+            baseFee,
+            blobBaseFee,
+            hash,
+            batcherHash,
+            operatorFeeScalar,
+            operatorFeeConstant
+        );
+
+        vm.prank(depositor);
+        (bool success,) = address(l1Block).call(calldataPacked);
+        require(success, "GasPriceOracleIsthmus_Test: Function call failed");
+    }
+
+    /// @dev Tests that `operatorFee` is set correctly.
+    function test_getOperatorFee_succeeds() external view {
+        assertEq(gasPriceOracle.getOperatorFee(10), 340); // 10 * 4 + 300
```

Comment:
unclear where the hidden constants are coming from unless you look at the top of the file. recommend referencing them, or redoing the math on them directly and comparing the results, makes it easy to match against the spec

Reply:
Fixed